Cargo.toml

```diff
@@ -22,7 +22,7 @@ panic = "unwind"
 # If your tests fail with SIGSEGV try and use 1.64 or newer.
 # Sadly, we can't just drop it because it's a workaround for
 # another issue https://github.com/tcdi/pgx/pull/208
-lto = "thin"
+#lto = "thin"
```

Comment:
why are we turning lto off?

Reply:
I keep accidentally committing it. Thanks for catching it!